cypress/fixtures/calendar_e2e_fixtures.js

```diff
@@ -0,0 +1,205 @@
+const currentDate = new Date();
+const currentYearInt = currentDate.getUTCFullYear();
+const currentYear = currentYearInt.toString();
+const currentMonth = ('0' + (currentDate.getUTCMonth() + 1).toString()).slice(-2);
+
+let febNumberOfDays;
+if ((currentYearInt % 4) === 0) {
+  if ((currentYearInt % 100) === 0) {
+    febNumberOfDays = (currentYearInt % 400) === 0 ? '29' : '28';
+  } else {
+    febNumberOfDays = '29';
+  }
+} else {
+  febNumberOfDays = '28';
+}
+
+
+const monthToLastDay = {
+  '01': '31',
+  '02': febNumberOfDays,
+  '03': '31',
+  '04': '30',
+  '05': '31',
+  '06': '30',
+  '07': '31',
+  '08': '31',
+  '09': '30',
+  '10': '31',
+  '11': '30',
+  '12': '31',
+};
+
+
+const lastDayOfMonth = monthToLastDay[currentMonth];
+
+const nextMonth = ('0' + ((currentDate.getUTCMonth() + 2) % 12).toString()).slice(-2);
+const nextMonthYearInt = currentDate.getUTCMonth() === 11 ? currentYearInt + 1 : currentYearInt;
+const nextMonthYear = nextMonthYearInt.toString();
+
+const lastDayOfNextMonth = monthToLastDay[nextMonth];
+
+export default {
```

Comment:
fixtures in cypress is about files that can be used inside tests

move this file in fragments/calendar and rename file please